operator/main.go

```diff
@@ -88,7 +88,7 @@ func init() {
 	flags.Int(option.ClusterIDName, 0, "Unique identifier of the cluster")
 	option.BindEnv(option.ClusterIDName)
```

Comment:
Do we need the same fix for `--cluster-id`?

Reply:
No, the problem is that the env variable set was `CILIUM_CILIUM_CLUSTER_NAME` by the `option.BindEnv` function